README.md

```diff
@@ -57,6 +57,18 @@ Apply `slo: "true"` under labels section of any alerting rule.
 ##### Note
 SLO alerts should be labeled with `severity: critical`
 
+### Labeling Control Plane alerts
+
+Since the SLO alerts would be routed further more to AppSRE in case of Data Plane SLO 
+alerts and to the Tiger team in case of Control Plane SLO alerts, we have to make sure 
```

Comment:
Tiger team ? 🐅
Info? Link?

Reply:
No info, no link yet as the team was not even formed.
We will wait for more details